Wire-iOS/Sources/Developer/DeveloperTools/DeveloperToolsViewModel.swift

```diff
@@ -180,6 +189,46 @@ final class DeveloperToolsViewModel: ObservableObject {
         DebugLogSender.sendLogsByEmail(message: "Send logs")
     }
 
+    private func checkRegisteredTokens() {
+        guard
+            let selfClient = selfClient,
+            let clientID = selfClient.remoteIdentifier,
+            let context = selfClient.managedObjectContext?.notificationContext
+        else {
+            return
+        }
+
+        let action = GetPushTokensAction(clientID: clientID) { result in
+            switch result {
+            case .success([]):
+                self.alertTitle = "No registered tokens"
+                self.alertBody = nil
+                self.isPresentingAlert = true
+
+            case let .success(tokens):
+                self.alertTitle = "Registered push tokens"
+                self.alertBody = tokens.map(\.debugDescription).joined(separator: "\n\n")
+                self.isPresentingAlert = true
+
+            case let .failure(error):
+                self.alertTitle = "Registered push tokens"
+                self.alertBody = "Failed to fetch push tokens: \(String(describing: error))"
+                self.isPresentingAlert = true
+            }
+        }
```

Comment:
move the 3 common lines out of switch case to here
```suggestion
                self.isPresentingAlert = true
        }
```

Reply:
done